### PR TITLE
Create Lightning-GPU 0.23 release version

### PR DESCRIPTION
**Pull request by AmintorDusko**
Release candidate v0.23 for pennylane-lightning-gpu.

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
 ### Improvements
+* Update builder and cuQuantum SDK support [(#10)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/10).
 
-### Documentation
+### Contributors
 
-### Bug fixes
+This release contains contributions from (in alphabetical order):
+Ali Asadi, and Lee James O'Riordan
```
Comment (maliasadi) [inline .github/CHANGELOG.md:9]: I think my name shouldn't be tagged in this one,

```suggestion
Lee James O'Riordan
```

**Review by maliasadi — COMMENTED** [review]
Thanks @AmintorDusko

File: .github/CHANGELOG.md (hunk 1 above)
Comment (mlxd) [reply .github/CHANGELOG.md:9]: Did you not add a commit or two to fix something in this release?

Comment (josh146) [reply .github/CHANGELOG.md:9]: @maliasadi even contributing via code review suggestions is a contribution that should be listed!

Comment (maliasadi) [reply .github/CHANGELOG.md:9]: Yes, I did, but they were minor. :slightly_smiling_face:

**Pull request closed without merging**